CHANGELOG.md

```diff
@@ -54,6 +54,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Change http code on create index API with bad input raising NotXContentException from 500 to 400 ([#4773](https://github.com/opensearch-project/OpenSearch/pull/4773))
 - Change http code for DecommissioningFailedException from 500 to 400 ([#5283](https://github.com/opensearch-project/OpenSearch/pull/5283))
 - Improve summary error message for invalid setting updates ([#4792](https://github.com/opensearch-project/OpenSearch/pull/4792))
+- Partially resolved issue #4687 by changing the improper status code for concurrent snapshot executions ([#5855](https://github.com/opensearch-project/OpenSearch/pull/5855))
```

Comment:
Let's explain what we did here? How about "Return 409 Conflict HTTP status instead of 500 on failure to concurrently execute snapshots"

Reply:
Understood, raising PR for this.